### ISSUE TITLE
Too much log spam

### ISSUE THREAD

**Issue by ensonic** [arduino/arduino-create-agent#194]
I get tons of messages like:
```
arduino-create-agent.desktop[1149]: time="2017-12-25T10:20:53+01:00" level=info msg="[]"
arduino-create-agent.desktop[1149]: time="2017-12-25T10:20:53+01:00" level=info msg="<nil>"
```
The default log level is hardcoded to INFO
https://github.com/arduino/arduino-create-agent/blob/devel/main.go#L143

Can this be a config setting and would it maybe make sense to move some to e.g. DEBUG.

**Comment by facchinm** [comment]
Agree, the log level should be configurable (quiet by default and configurable via a flag). Since we are still under heavy development it's quite handy to have a very verbose log so it becomes easier for the user to file good bugreports.
